File: CodeWars_Ruby/max_and_min.rb
#!/usr/bin/env ruby

def min(list)
  list = list.sort
  list[0]
end

def max(list)
  list = list.sort
  list[-1]
end

puts min([-52, 56, 30, 29, -54, 0, -110])
puts min([42, 54, 65, 87, 0])
puts max([4, 6, 2, 1, 9, 63, -134, 566])
puts max([5])

File: CodeWars_Python/max_and_min.py
#!/usr/bin/env python3


def min(arr):
    arr.sort()
    return arr[0]


def max(arr):
    arr.sort()
    return arr[-1]


print(min([-52, 56, 30, 29, -54, 0, -110]))
print(min([42, 54, 65, 87, 0]))
print(min([1, 2, 3, 4, 5, 10]))
print(min([-1, -2, -3, -4, -5, -10]))
print(min([9]))
